public/js/grid/react_util.js

```diff
@@ -0,0 +1,115 @@
+var CoursePanel = React.createClass({
+    render: function() {
+        return (
+            <ul className="trapScroll-enabled" id="course-select">
+                <li id="clear-all">
+                    <h3>Clear All</h3>
+                </li>
+            </ul>
+        );
+    }
+});
+
+var SearchPanel = React.createClass({
+    render: function() {
+        return (
+            <div>
+                <div id="filter-container">
+                    <form onsubmit="return false;">
```

Comment:
Never mind. In order to get the search icon to show up properly, you'd need either the form or another wrapper div (otherwise it would just be used as a background image for the input box)